src/MSL/string.c

```diff
@@ -1,5 +1,10 @@
+#include <MSL/string.h>
 #define EOF -1
+
+#ifdef MWERKS_GEKKO
```

Comment:
```suggestion
#ifdef __MWERKS__
```
Not gekko-specific?

Reply:
[p. 267](https://www.nxp.com/docs/en/reference-manual/CWMCUCFCMPREF.pdf)